homeassistant/components/abode/const.py

```diff
@@ -6,5 +6,4 @@
 DOMAIN = "abode"
 ATTRIBUTION = "Data provided by goabode.com"
 
-DEFAULT_CACHEDB = "abodepy_cache.pickle"
```

Comment:
Should we try to remove this file?

Reply:
It doesn't do any harm to leave it. The library just doesn't use it any more.